stdlib/2/user.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for user (Python 2)
+
+# Docs: https://docs.python.org/2/library/user.html
+# Source: https://hg.python.org/cpython/file/2.7/Lib/user.py
+from typing import BinaryIO, Text, Union
+
+
+home: str
+pythonrc: Union[bytes, Text]
```

Comment:
In this case yes, but it can have multiple [return both](https://github.com/python/typeshed/blob/master/stdlib/2/os/path.pyi#L62) in general.